ethereum/contracts/interfaces/IERC4906.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: Unlicense
```

Comment:
OZ has the contract but it's not available via npm, yet. https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/interfaces/IERC4906.sol.

Since the Rigs contract already inherits from those that implement IERC165 and IERC721, I didn't have IERC4906 inherit them here (double checking this is fine as it differs from the contract linked above).